python/ray/tune/BUILD

```diff
@@ -94,15 +94,6 @@ py_test(
     tags = ["team:ml", "exclusive", "rllib"],
 )
 
-py_test(
-    name = "test_cluster_searcher",
```

Comment:
This test is pretty much replaced by `test_experiment_restore`. The test setup was strange and hard to fix.